packages/block-editor/src/components/warning/index.js

```diff
@@ -29,19 +34,16 @@ function Warning( { className, actions, children, secondaryActions } ) {
 								</span>
 							) ) }
 						{ secondaryActions && (
-							<Dropdown
+							<DropdownMenu
 								className="block-editor-warning__secondary"
-								contentClassName="block-editor-warning__dropdown"
-								position="bottom left"
-								renderToggle={ ( { isOpen, onToggle } ) => (
-									<Button
-										icon={ moreHorizontal }
-										label={ __( 'More options' ) }
-										onClick={ onToggle }
-										aria-expanded={ isOpen }
-									/>
-								) }
-								renderContent={ () => (
+								icon={ <BlockIcon icon={ moreHorizontal } /> }
```

Comment:
Why `BlockIcon`, can we use the svg `moreHorizontal` directly as `icon`? If not, I think it's better to use `Icon from '@wordpress/icons` component.

Reply:
We can. I had the impression the usage of `BlockIcon` was preferred from some other code I had seen in the past.. Will investigate more the differences of them.

Changed it. Thanks!

Reply:
I think BlockIcon is for "block" icons (icons used for block registration). Block icons support some color properties there...

Reply:
Yes you're right! And internally uses the `Icon` component as you proposed. 💯